### DISCUSSION TITLE
Is there any config in openupgrade to disable oca install dependency?

### DISCUSSION THREAD

**Discussion by ronny-montano** [OCA/OpenUpgrade#4416] [Q&A]
**Is your feature request related to a problem?**
Is there any config in openupgrade to disable oca install dependency?

**Describe alternatives you've considered**
The point is that it has performed a migration using openupgrade and as a result it has installed many OCA addons that I did not have.

**Comment by ronny-montano** [comment]
@pedrobaeza please.. ! can you help me?

**Comment by pedrobaeza** [comment]
Unless you are more explicit, I can't help.

**Comment by ronny-montano** [comment]
> Unless you are more explicit, I can't help.

I have migrated a db from v13 to v16, and in the final db, I have obtained the db with several OCA addons that do not exist in v13 installed.
v13 256 addons installed
v16 1562 addons installed

**Comment by pedrobaeza** [comment]
That's not a problem of OpenUpgrade. You are clearly doing something wrong. Are you sure they are installed? Or they are just listed? Anyway, as said, this is not done by OpenUpgrade. And this shouldn't be a issue, so converting it to discussion.

**Comment by ronny-montano** [comment]
I didn't say it was an openupgrade error, just that this happened to me, as a result of the migration, and I wanted to know if there was any option to prevent it from happening.
